host/OEHOSTMR/CMakeLists.txt

```diff
@@ -0,0 +1,189 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+# Add a library oehostmr independent of libsgx_enclave_common/libsgx_dcap_ql
+if (UNIX)
+  set(PLATFORM_HOST_MR_SRC
+    crypto/openssl/init.c
+    crypto/openssl/key.c
+    crypto/openssl/rsa.c
+    crypto/openssl/sha.c
+	linux/hostthread.c
```

Comment:
Kindly fix the spacing issues in this file.

Reply:
Thanks, tab is replace with space.